catalog/solr/catalog-solr-external-provider/src/main/java/ddf/catalog/solr/external/SolrHttpCatalogProvider.java

```diff
@@ -253,14 +259,21 @@ public void updateServer(String urlValue) {
         }
     }
 
+    private void updateServer() {
+        serverFuture = SolrServerFactory.getHttpSolrServer(url,
+                SOLR_CATALOG_CORE_NAME,
+                SOLR_CATALOG_CONFIG_FILE);
+        server = null;
+    }
+
     private CatalogProvider getProvider() {
         if (firstUse) {
```

Comment:
The `firstUse` bool has been there for longer than the repository (pre 8/15). 
I'll take a whack at rewriting it.